src/search/landmarks/exploration.h

```diff
@@ -50,12 +42,12 @@ struct ExUnaryOperator {
     int base_cost; // 0 for axioms, 1 for regular operators
 
     int unsatisfied_preconditions;
-    int h_add_cost;
+    bool excluded;
     int h_max_cost;
-    int depth;
     ExUnaryOperator(const std::vector<ExProposition *> &pre, ExProposition *eff,
                     int op_or_axiom_id, int base)
-        : op_or_axiom_id(op_or_axiom_id), precondition(pre), effect(eff), base_cost(base) {}
+        : op_or_axiom_id(op_or_axiom_id), precondition(pre), effect(eff),
+          base_cost(base), excluded(false) {}
```

Comment:
`base` is a bad name for `operator cost`. But I expect this will go away soon anyway. And I suppose it's not relevant to this issue.